cluster/okd-on-ovirt/roles/okd-on-ovirt/tasks/wait_cluster.yaml

```diff
@@ -0,0 +1,37 @@
+- name: query assisted OKD cluster status until ready
+
+  block:
+  
+  - name: create cluster via  POST
+    uri:
+      url: "http://{{ assisted_fqdn }}:8080/api/assisted-install/v2/clusters/{{okd_cluster_id}}"
+      method: GET
+      headers:
+        Version: "4"
+        Content-Type: "application/json"
+        Accept: "application/json"
+      status_code: 200
+    register: cluster_info
```

Comment:
Think that instead of going recursive and waiting you can use the `until`, `retries`, `delay`